router/package_request_parts.go

```diff
@@ -13,8 +13,10 @@ import (
 const (
 	at                          = '@'
 	dot                         = '.'
+	hyphen                      = '-'
 	slash                       = '/'
 	shaLength                   = 40
+	shortSHALength              = 6
```

Comment:
I tried 5 but it was more consistent with 6 characters. Can try again though. Also will change to `minShortSHALength.`